class UserNotFoundException(Exception):
    pass


class UserAlreadyExistsException(Exception):
    pass


class GroupNotFoundException(Exception):
    pass
